tests/test_install.py

```diff
@@ -99,6 +99,74 @@ def test_install_mixed_case2(selenium_standalone_micropip, jinja2):
     )
 
 
+PYTZ_2020_5_WHEEL = "pytz-2020.5-py2.py3-none-any.whl"
+PYTZ_2020_5_URL = f"https://files.pythonhosted.org/packages/89/06/2c2d3034b4d6bf22f2a4ae546d16925898658a33b4400cfb7e2c1e2871a3/{PYTZ_2020_5_WHEEL}"
```

Comment:
Let's not use the external URL in the unittest.

You can put the wheel file in the [test directory](https://github.com/pyodide/micropip/tree/main/tests/test_data/wheel), and use the [wheel_catalog](https://github.com/pyodide/micropip/blob/5ce6e1f26f0680bf7db5f56bb9c98cbbe3726908/tests/conftest.py#L160C5-L160C18) fixture to generate a URL for the wheel. ([example](https://github.com/pyodide/micropip/blob/5ce6e1f26f0680bf7db5f56bb9c98cbbe3726908/tests/test_list.py#L45-L46))

Reply:
hm, the wheel catalog is keyed by name, which makes it not great for testing sensitivity to version resolution. i'll see about the other stuff, and swing back to this,

Reply:
using the pre-existing `pytest` fixture